Dockerfile

```diff
@@ -80,20 +89,15 @@ ENV PATH=${cmake_path}:$PATH
 
 COPY scripts scripts
 
-COPY ./irods-dev_4.3.0-1~focal_amd64.deb ./irods-runtime_4.3.0-1~focal_amd64.deb /irods43X_packages/
-
-RUN bash scripts/get_build_deps.sh
-
 RUN mkdir irods_s3_bridge
 
 COPY . irods_s3_bridge
 
 RUN bash scripts/build_bridge.sh
 
-RUN ls -l /irods_s3_bridge/build && dpkg -i /irods_s3_bridge/build/irods-experimental-client-s3-api_0.1.0-1~focal_amd64.deb
-
-RUN chmod +x scripts/run_bridge.sh
+RUN ls -l /irods_s3_bridge/build && dpkg -i /irods_s3_bridge/build/irods-experimental-client-s3-api_0.1.0-0~focal_amd64.deb
 
-RUN mkdir -p /root/.irods
+RUN chmod +x scripts/run_bridge.sh && \
```

Comment:
Don't need to do this right now, but we could possibly set the +x on the actual files in the repo to avoid this step and/or use the fancy `--chmod` option for `COPY` and copy in individual files: https://docs.docker.com/engine/reference/builder/#copy

Reply:
Created #56 for this.